packages/calcite-components/src/components/dropdown-group/dropdown-group.tsx

```diff
@@ -45,6 +42,13 @@ export class DropdownGroup extends LitElement {
   /** Specifies and displays a group title. */
   @property({ reflect: true }) groupTitle: string;
 
+  /**
+   * The position of the group in the dropdown menu.
+   *
+   * @internal
+   */
+  @property() position: number = -1;
```

Comment:
Should there be a `defaults` test to test for `-1` now that its not undefined by default?

Reply:
We can skip it since it's an implementation detail.